### PR TITLE
add labels section in docker options

### PR DESCRIPTION
**Pull request by laverya**
[ch10756]

### CODE REVIEW

File: content/docs/native/packaging-an-application/docker-options.md [hunk 1]
```diff
@@ -118,26 +125,38 @@ Since setting ulimit settings in a container requires extra privileges not avail
...
     dynamic: true
 ```
+
+{{< linked_headline "Labels" >}}
+
+[Labels](https://docs.docker.com/config/labels-custom-metadata/) can be applied to a container by Replicated. Labels are templateable, and will be split on the first `=` to form the label key and value. If no `=` is present, the entire string will become the key and the value will be the empty string.
+
+For example, `my.container.label` would become a label with a key of `my.container.label` and an empty value while `my.container.value=IMPORTANT=FALSE` would have a key of `my.container.value` and a value of `IMPORTANT=FALSE`.
+
+If a container has the label `com.replicated.excludelogs=true` set, Replicated will not include its logs in Support Bundles.
```
Comment (emosbaugh) [inline content/docs/native/packaging-an-application/docker-options.md]: i think you should omit this from here

File: content/docs/native/packaging-an-application/docker-options.md [hunk 2]
```diff
@@ -118,26 +125,38 @@ Since setting ulimit settings in a container requires extra privileges not avail
...
+
+```yaml
+    labels: ["com.replicated.excludelogs=true"]
```
Comment (emosbaugh) [inline content/docs/native/packaging-an-application/docker-options.md]: use a different example

Comment (laverya) [reply content/docs/native/packaging-an-application/docker-options.md]: Will do

File: content/docs/native/packaging-an-application/docker-options.md (hunk 1 above)
Comment (laverya) [reply content/docs/native/packaging-an-application/docker-options.md]: Sure

**Pull request merged**